### PR TITLE
Single source the docs lints

### PR DESCRIPTION
**Pull request by kagarmoe**
Signed-off-by: kagarmoe <kimberly.garmoe@progress.com>

## Description

[Please describe what this change achieves]

## Issues Resolved

[List any existing issues this PR resolves, or any Discourse or
StackOverflow discussion that's relevant]

## Check List

- [ ] Spell Check
- [ ] Local build
- [ ] Examine the local build
- [ ] All tests pass

### CODE REVIEW

**Comment by IanMadd** [comment]
Is the sync-config.yml going to do anything? I thought it only used the config.yml.

**Comment by kagarmoe** [comment]
I doubt it but I didn't wanted to gather my thoughts outside of the config.yml to start with.

File: .expeditor/config.yml [hunk 1]
```diff
@@ -40,3 +40,18 @@ subscriptions:
   - workload: chef/cookstyle:main_completed:project_promoted:chef/cookstyle:main:*
     actions:
     - bash:.expeditor/update_hugo_modules_cookstyle.sh
+  - workload: pull_request_merged:inspec/inspec-aws:main:*
+    actions:
+    - bash:.expeditor/update_hugo_modules_pull_request_merged.sh:
+        only_if_modified:
+          - docs-chef-io/static/release-notes/inspec-aws/*
+    - workload: pull_request_merged:inspec/inspec-azure:main:*
```
Comment (mimaslanka) [inline .expeditor/config.yml]: back space to line up with above workload

File: .expeditor/config.yml [hunk 2]
```diff
@@ -40,3 +40,18 @@ subscriptions:
...
+      actions:
+    - bash:.expeditor/update_hugo_modules_pull_request_merged.sh:
+        only_if_modified:
+          - docs-chef-io/static/release-notes/inspec-azure/*
+    - workload: pull_request_merged:chef/chef-web-docs:main:*
```
Comment (mimaslanka) [inline .expeditor/config.yml]: same as above

**Review by IanMadd — APPROVED** [review]

**Pull request merged**